prow/plugins/bugzilla/bugzilla_test.go

```diff
@@ -626,6 +626,8 @@ Instructions for interacting with me using PR comments are available [here](http
 			expectedLabels: []string{"bugzilla/valid-bug"},
 			expectedComment: `org/repo#1:@user: This pull request references [Bugzilla bug 123](www.bugzilla/show_bug.cgi?id=123), which is valid.
 
+No validations were run on this bug.
```

Comment:
I would expect at least one complicated test case in this which has a long list.